src/index.ts

```diff
@@ -1,13 +1,30 @@
 interface CachedData {
   src: string
-  data: Promise<string | Error>
+  data: Promise<string | CSPTrustedHTMLToStringable | Error>
 }
 const privateData = new WeakMap<IncludeFragmentElement, CachedData>()
 
 function isWildcard(accept: string | null) {
   return accept && !!accept.split(',').find(x => x.match(/^\s*\*\/\*/))
 }
 
+// CSP trusted types: We don't want to add `@types/trusted-types` as a
+// dependency, so we use the following types as a stand-in.
+interface CSPTrustedTypesPolicy {
+  createHTML: (s: string, response: Response) => CSPTrustedHTMLToStringable
+}
+// Note: basically every object (and some primitives) in JS satisfy this
+// `CSPTrustedHTMLToStringable` interface, but this is the most compatible shape
+// we can use.
+interface CSPTrustedHTMLToStringable {
+  toString: () => string
+}
+let cspTrustedTypesPolicyPromise: Promise<CSPTrustedTypesPolicy> | null = null
+// Passing `null` clears the policy.
+export function setCSPTrustedTypesPolicy(policy: CSPTrustedTypesPolicy | Promise<CSPTrustedTypesPolicy> | null): void {
```

Comment:
I still think this should be a static member on `IncludeFragmentElement`, as the API would be more apparent as to intent:

```js
import IncludeFragmentElement from "include-fragment-element"
import {policy} from 'wherever'

// We know that this is only setting the policy _for_ `IncludeFragmentElement` and not globally.
IncludeFragmentElement.setCSPTrustedTypesPolicy(policy);

Reply:
Added in https://github.com/github/include-fragment-element/pull/81/commits/672e8369eacb1138a11ef8faa32a7695ca7ee764!